hadoop-ozone/common/src/main/java/org/apache/hadoop/ozone/om/protocol/OzoneManagerProtocol.java

```diff
@@ -908,4 +909,18 @@ default PrepareStatusResponse getOzoneManagerPrepareStatus(long txnId)
   default CancelPrepareResponse cancelOzoneManagerPrepare() throws IOException {
     return CancelPrepareResponse.newBuilder().build();
   }
+
+  /**
+   * Send RPC request with or without payload to OM
+   * to benchmark RPC communication performance.
+   * @param payloadReq payload in request.
+   * @param payloadSizeResp payload size of response.
+   * @param emptyResp if the payload of response is empty or not.
+   * @throws IOException if there is error in the RPC communication.
+   * @return EchoRPCResponse.
+   */
+  EchoRPCResponse echoRPCReq(byte[] payloadReq,
+                             int payloadSizeResp, boolean emptyResp)
```

Comment:
I think this API can be simplified, if the response is empty the `payloadSizeResp` can be 0 and we can drop the boolean.

Reply:
yes sure!